docs/user/metrics/features.csv

```diff
@@ -8,5 +8,6 @@ Library,ML Backend,I/O Backend,Spatial Backend,Transform Backend,Datasets,Weight
 `TerraTorch`_,PyTorch,"GDAL, h5py, pandas, xarray",R-tree,Albumentations,22,1,✅,✅,❌,🚧
 `SITS`_,R Torch,GDAL,-,tidyverse,22,0,❌,✅,✅,✅
 `scikit-eo`_,"scikit-learn, TensorFlow","pandas, scipy, numpy, rasterio","geopandas",numpy,0,0,❌,❌,❌,🚧
+`geo-bench`_,Pytorch,"h5py,pandas",-,Kornia,12,-,❌,❌,❌,❌
```

Comment:
* **ML Backend**: Pytorch -> PyTorch
* **I/O Backend**: I also see GDAL (via rasterio), tifffile, and maybe scipy depending on what it's used for
* **Transform Backend**: I don't see Kornia listed in pyproject.toml, is this via TorchGeo?
* **Datasets**: I see at least 15 in https://github.com/ServiceNow/geo-bench/tree/main/make_benchmark/dataset_converters, are these still a WIP?

Reply:
- datasets were transformed to h5 files
- transforms are defined in the experiment version of geo-bench
- there are 12 datasets in the paper

Reply:
We aren't basing # datasets on the paper, we're basing it on the code. This number will change over time. If GEO-Bench 1 + 2 is 30 datasets, and all are concurrently supported by the repo, then the repo has 30 datasets.

Reply:
12 are supported by the code currently